x/auth/signing/direct/direct.go

```diff
@@ -0,0 +1,66 @@
+package direct
+
+import (
+	"fmt"
+
+	signingtypes "github.com/cosmos/cosmos-sdk/types/tx/signing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	types "github.com/cosmos/cosmos-sdk/types/tx"
+	"github.com/cosmos/cosmos-sdk/x/auth/signing"
+)
+
+// ProtoTx defines an interface which protobuf transactions must implement for
+// signature verification via SignModeDirect
+type ProtoTx interface {
+	// GetBodyBytes returns the raw serialized bytes for TxBody
+	GetBodyBytes() []byte
+
+	// GetBodyBytes returns the raw serialized bytes for AuthInfo
+	GetAuthInfoBytes() []byte
+}
+
+// ModeHandler defines the SIGN_MODE_DIRECT SignModeHandler
+type ModeHandler struct{}
+
+var _ signing.SignModeHandler = ModeHandler{}
+
+// DefaultMode implements SignModeHandler.DefaultMode
+func (h ModeHandler) DefaultMode() signingtypes.SignMode {
```

Comment:
could we omit the receiver identifier `h` here too?

Reply:
we need this as ModeHandler has to satisfy SignModeHandler interface

Reply:
This is just about removing `h` here which makes sense.